### PR TITLE
Minor tweaks to CI

### PR DESCRIPTION
**Pull request by crusaderky**
Some very minor touches to the CI scripts

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *.pyc
+*.pyo
```
Comment (jakirkham) [inline .gitignore]: Common thing to do here is [`*.py[cod]`]( https://github.com/github/gitignore/blob/1fb9ea045e619c61e223371533cdea118efbea5d/Python.gitignore#L3 ). Also would be good to add `__pycache__/` while we are at it.

**Comment by jakirkham** [comment]
Looks fine to me. Made a minor suggestion above.

**Comment by crusaderky** [comment]
@jakirkham changed as requested

**Comment by jakirkham** [comment]
Thanks @crusaderky :)

**Pull request merged**